test/org/sagebionetworks/bridge/dynamodb/DynamoTaskTest.java

```diff
@@ -88,8 +102,8 @@ public void hasValidStatusBasedOnTimestamps() throws Exception {
         assertEquals(TaskStatus.DELETED, task.getStatus());
         
         task = new DynamoTask();
-        task.setScheduledOn(DateTime.now().minusHours(1).getMillis());
-        task.setExpiresOn(DateTime.now().plusHours(1).getMillis());
+        task.setLocalScheduledOn(now.minusHours(1));
+        task.setLocalExpiresOn(now.plusHours(1));
         assertEquals(TaskStatus.AVAILABLE, task.getStatus());
     }
     
```

Comment:
Can DynamoTask ever not have a timezone? Is it worth adding a few basic tests for those cases?